pandas/core/frame.py

```diff
@@ -5932,14 +5932,23 @@ def _construct_result(self, result) -> "DataFrame":
     The resulting index will be a MultiIndex with 'self' and 'other'
     stacked alternately at the inner level.
 
+Raises
+------
+ValueError
+    * When both DataFrames don't have identical labels or shape.
```

Comment:
First, the formatting looks a bit different to other docstrings, e.g. in pandas/core/algorithms.py we find

```
    Raises
    ------
    IndexError
        When `indices` is out of bounds for the array.
    ValueError
        When the indexer contains negative values other than ``-1``
        and `allow_fill` is True.
```

, is the bullet point `* ` like this used elsewhere in the codebase?

Second, I think the usage of "both" here isn't usual, how about:

```
ValueError
    When the two DataFrames don't have identical labels or shape.
```

Reply:
Yes, first I had two raises, so had bullet points, forgot to remove later. Adjusted this.